go/src/koding/klientctl/template.go

```diff
@@ -151,6 +151,22 @@ func TemplateDelete(c *cli.Context, log logging.Logger, _ string) (int, error) {
 }
 
 func TemplateInit(c *cli.Context, log logging.Logger, _ string) (int, error) {
+	output := c.String("output")
+
+	if _, err := os.Stat(output); err == nil {
+		yn, err := helper.Ask("Do you want to overwrite %q file? [y/N]: ", output)
+		if err != nil {
+			return 1, err
+		}
+
+		switch strings.ToLower(yn) {
+		case "yes", "y":
+			fmt.Fprintln(os.Stderr)
```

Comment:
because `helper.Ask` prints to stdout and then we print to stderr even when we have no errors. This may be confusing when someone eg. redirects only stderr.